.github/workflows/pr.yml

```diff
@@ -0,0 +1,19 @@
+# The Licensed Work is (c) 2022 Sygma
+# SPDX-License-Identifier: BUSL-1.1
```

Comment:
shouldn't the license be LGPL-3.0-only?

Reply:
Oops, copy pasted a lot of stuff :sweat_smile: